opengrok-indexer/src/main/java/org/opengrok/indexer/history/SCCSRepository.java

```diff
@@ -62,6 +63,12 @@ public class SCCSRepository extends Repository {
     @VisibleForTesting
     static final String CODEMGR_WSDATA = "Codemgr_wsdata";
 
+     /**
+     * This is a static replacement for 'working' field. Effectively, check if hg is working once in a JVM
+     * instead of calling it for every SCCSRepository instance.
+     */
+    private static final Supplier<Boolean> HG_IS_WORKING = LazilyInstantiate.using(SCCSRepository::isHgWorking);
```

Comment:
this variable should probably by named differently to match the repository implementation (fix copy-n-paste programming).